server/handlers/comment.js

```diff
@@ -8,109 +8,87 @@ import {
 } from './find';
 import { deleteNotification } from './notification';
 
-export const getCommentList = (req, res) =>
-  new Promise(() => {
-    getList()
-      .then(data => {
-        if (!data) {
-          return res.send(data);
-        } else {
-          const commentList = _.map(data, doc => ({
-            commentId: doc.id,
-            userHandle: doc.userHandle,
-            postId: doc.postId,
-            body: doc.body,
-            createdAt: doc.createdAt,
-          }));
-          return res.send(commentList);
-        }
-      })
-      .catch(err => {
-        console.error(err);
-        res.status(500).send(err);
-      });
+/**
+ * Attempts to retrieve a full comment list
+ * @type {RouteHandler}
+ */
+export const getCommentList = async (req, res) => {
+  const data = await getList().catch(err => {
+    res.status(500);
```

Comment:
if all you're going to do is 500, you should just plop something at the top level router that 500's on errors.

Reply:
Fixed